<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GW250Widget</class>
 <widget class="QWidget" name="GW250Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Menu</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="MenuWidget" name="menuPage" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Trouble Code</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="TroubleCodeWidget" name="troubleCodePage" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Live Data</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="LiveDataWidget" name="liveDataPage" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MenuWidget</class>
   <extends>QWidget</extends>
   <header>menuwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TroubleCodeWidget</class>
   <extends>QWidget</extends>
   <header>troublecodewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LiveDataWidget</class>
   <extends>QWidget</extends>
   <header>livedatawidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
